docs/index.md

```diff
@@ -1,97 +1,109 @@
+   - Owner:                #The organization, user or project that this repo will belong to
+   
+   - Repository: `labs-<Component>`     #The name of the repository
+   
+    
 2. Create Helm values file with following contents(this is only needed in the lab as the application will be run on sandbox).
 
    CFT: values.sandbox.template.yaml
-   
-   
+
+    ```yaml
         java:
           # Don't modify below here
           image: ${IMAGE_NAME}
           ingressHost: ${SERVICE_FQDN}
+    ```
    
 3. Get the Pull Request reviewed and merged.
 
 #### Build application
 
-1. Login to Jenkins and select "HMCTS - Labs" folder. Scan the organization by clicking on `Scan Organization Now`. New repository should be listed under repositories after scan finishes. Logs can be monitored under `Scan Organization Log`. Any GitHub repository that starts with `labs*` will be listed as part of this scan.
-
-   CFT: [jenkins](https://sandbox-build.platform.hmcts.net/job/HMCTS_LABS/)
+1. Login to Jenkins and select ["HMCTS - Labs"](https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_LABS/) folder. Scan the organization by clicking on `Scan Organization Now`. New repository should be listed under repositories after scan finishes. Logs can be monitored under `Scan Organization Log`. Any GitHub repository that starts with `labs*` will be listed as part of this scan.
 
 
 2. Run the jenkins pipeline against the `master` branch.
 
 #### Configure load balancing for HA
 
-1. We load balance across AKS clusters using `Azure Application Gateway`. Add a couple of lines of config for the application.
-
-   CFT:  [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml)
+1. We load balance across AKS clusters using `Azure Application Gateway`. Add a couple of lines of config for the application in [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml).
 
+   ```yaml
+   #labs
+      - product: "labs"
+        component:     #githubreponame without "labs" prefix
+   ```
      
-           #labs
-               - product: "labs"
-                 component:     #githubreponame without "labs" prefix
-   
 #### Deploy application
 
 1. We practise [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
-
-   CFT: Read the documentation and follow the instructions [here]( https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md)
+   Read the [documentation]( https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md) and follow the instructions.
 
 #### Access application
 
 1. Access the deployed application using the URL.
 
+   ```
+   http://<product>-<component>-sbox.service.core-compute-sandbox.internal 
+   ```  
    
-      http://<product>-<component>-sbox.service.core-compute-sandbox.internal   
-   
-#### Customize application
+#### Customise application
```

Comment:
this needs a lot more detail, we want to take the user through making a code change which users their config and see it displayed